libs/testing/cypress/node/src/lib/plugins/auth.ts

```diff
@@ -1,20 +1,32 @@
 import { auth } from '../testing-cypress';
+import { USER_FIXTURES_PASSWORD } from '@blockframes/devops';
 
-export async function validateAuthUser(email: string) {
-  const user = await auth.getUserByEmail(email);
-  return await auth.updateUser(user.uid, { emailVerified: true });
+export async function createUser(data: { uid: string; email: string }) {
+  return await auth.createUser({ ...data, password: USER_FIXTURES_PASSWORD });
 }
 
-export async function getAuthUserByEmail(email: string) {
+export async function getUser(emailOrUid: string) {
   try {
-    return await auth.getUserByEmail(email);
+    if (emailOrUid.includes('@')) return await auth.getUserByEmail(emailOrUid);
+    return await auth.getUser(emailOrUid);
```

Comment:
```suggestion
    return emailOrUid.includes('@')) ? await auth.getUserByEmail(emailOrUid) : await auth.getUser(emailOrUid);
```

uid can never contain `@` ?

Reply:
Only if I create it myself, and I won't :)